changelog.md

```diff
@@ -19,9 +46,30 @@
    * Elvish Enchantress - decreased ranged slow attack from 7×4 to 6×4, decreased ranged magical attack from 13×4 to 11×4, decreased cost from 70 to 62, decreased XP requirement from 198 to 180
    * Elvish Sylph - decreased ranged magical attack from 16×5 to 13×5, decreased cost from 161 to 135
    * Added new Fire Wisp unit
+   * Added portraits for Skeletal Rider, Wyvern, TSG’s Eyestalk, TRoW’s vampire lady, and an alternative portrait for bears
  ### User interface
+   * Added a button to the build info dialog to rerun the 1.16 to 1.18 migration tool (issue #7936)
+   * Use a gold completion laurel for campaigns with only one difficulty setting
+   * Refined the layout of the Load Game dialog
+   * New texture-based minimap rendering
+   * Added a "Teleport Unit" option to the debug menu
  ### WML Engine
+   * Fixed the Ambush ability not working in Bluff/Glutch Forest terrain
+   * Multiple fixes to replay handling
+   * Image Path Functions now accept percentages for scaling sizes
+   * The AMLAs that a unit has are now accessible via WFL
  ### Miscellaneous and Bug Fixes
+   * Remove special handling of config directory on Linux, use the userdata folder as on Windows and MacOS (#8848)
+   * Added new `color_range` palettes `yellow` and `pink`
+   * Added environment variable `WESNOTH_NO_LOG_FILE`, equivalent to the `--no-log-to-file` command line flag
+   * Fixed attack calculations when resistances counter `[damage_type]alternative_type=`
+   * Fixed replays marking achievements as complete (issue #8858)
+   * Fixed build with Boost 1.85
+   * Fixed a crash in the recall dialog when turning off all sorting options (issue #8878)
+   * Fixed clicking on a trait in the sidebar sometimes opening the wrong trait’s help page
+   * Removed wesnothd’s unused threads command line options
+   * Fixed a bug causing excessive pixellation of scaled images
+   * General cleanup of unnecessary C++ header file includes
```

Comment:
Imo this last point (cleanup includes) is really not important enough to be mentioned here, since it not relevant for any user of the game.

Reply:
The changelog is not just for players. There is a good chance more builds than have already been fixed are broken because of the include cleanup so it might help someone get an idea of what's going on.